stream.go

```diff
@@ -112,6 +112,12 @@ func newClientStream(ctx context.Context, desc *StreamDesc, cc *ClientConn, meth
 		put func()
 	)
 	c := defaultCallInfo
+	if sc, ok := cc.getMethodConfig(method); ok {
+		c.failFast = !sc.WaitForReady
```

Comment:
- wait_for_ready: overriding is done in line#124

- timeout: this is handled by the standard context package already -- when there are 2 context.WithTimeout calles, the valid one is the mininal.